packages/debug/libunwind/package.mk

```diff
@@ -2,21 +2,26 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libunwind"
-PKG_VERSION="1.4.0"
-PKG_SHA256="df59c931bd4d7ebfd83ee481c943edf015138089b8e50abed8d9c57ba9338435"
+PKG_VERSION="1.6.0"
+PKG_SHA256="7194c20dda841b26917c0579e6467b53f7ad2e3f9452e5c77f3c3829c4656b70"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.nongnu.org/libunwind/"
 PKG_URL="http://download.savannah.nongnu.org/releases/libunwind/libunwind-${PKG_VERSION}.tar.gz"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="library to determine the call-chain of a program"
 PKG_BUILD_FLAGS="+pic"
+PKG_TOOLCHAIN="autotools"
 
 PKG_CONFIGURE_OPTS_TARGET="--enable-static \
                            --disable-shared \
                            --disable-minidebuginfo \
                            --disable-documentation \
                            --disable-tests"
 
+pre_configure_target() {
+  :
+}
+
```

Comment:
Why is this empty function needed?

Reply:
Thanks for the pickup. Hangover from my squash.